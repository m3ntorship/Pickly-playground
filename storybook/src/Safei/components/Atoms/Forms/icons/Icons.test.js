import React from "react";
import renderer from "react-test-renderer";
import ArrowDown from "./ArrowDown";
import CorrectIcon from "./Correct";
import WrongIcon from "./Wrong";
import Home from "./Home";

describe("test for my icons", () => {
  it("display arrow down icon comp", () => {
    const tree = renderer.create(<ArrowDown />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("display home icon comp", () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("display the wrong icon comp", () => {
    const tree = renderer.create(<WrongIcon />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("display the correct icon comp", () => {
    const tree = renderer.create(<CorrectIcon />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
