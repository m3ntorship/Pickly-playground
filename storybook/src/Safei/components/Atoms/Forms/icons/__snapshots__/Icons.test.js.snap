// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for my icons display arrow down icon comp 1`] = `
<svg
  className="w-3 h-3"
  fill="#8D9A9E"
  viewBox="0 0 20 20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clipRule="evenodd"
    d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
    fillRule="evenodd"
  />
</svg>
`;

exports[`test for my icons display home icon comp 1`] = `
<svg
  className="w-3 h-3"
  fill="none"
  viewBox="0 0 14 14"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0.999801 7.66669H1.66647V9.00002V12.3334C1.66647 13.0687 2.26447 13.6667 2.9998 13.6667H4.9998H8.9998H10.9998C11.7351 13.6667 12.3331 13.0687 12.3331 12.3334V9.00002V7.66669H12.9998C13.2691 7.66669 13.5131 7.50402 13.6158 7.25536C13.7191 7.00602 13.6618 6.71936 13.4711 6.52869L7.47113 0.528691C7.21047 0.268024 6.78913 0.268024 6.52847 0.528691L0.528467 6.52869C0.337801 6.71936 0.280467 7.00602 0.383801 7.25536C0.486467 7.50402 0.730467 7.66669 0.999801 7.66669ZM5.66647 12.3334V9.00002H8.33313V12.3334H5.66647ZM6.9998 1.94269L10.9998 5.94269V9.00002L11.0005 12.3334H9.66647V9.00002C9.66647 8.26469 9.06847 7.66669 8.33313 7.66669H5.66647C4.93113 7.66669 4.33313 8.26469 4.33313 9.00002V12.3334H2.9998V9.00002V7.00002V5.94269L6.9998 1.94269Z"
    fill="#5B6366"
  />
</svg>
`;

exports[`test for my icons display the correct icon comp 1`] = `
<svg
  className="w-3 h-3"
  fill="#07B255"
  viewBox="0 0 20 20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clipRule="evenodd"
    d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z"
    fillRule="evenodd"
  />
</svg>
`;

exports[`test for my icons display the wrong icon comp 1`] = `
<svg
  className="w-3 h-3"
  fill="#FE544A"
  viewBox="0 0 20 20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clipRule="evenodd"
    d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-7 4a1 1 0 11-2 0 1 1 0 012 0zm-1-9a1 1 0 00-1 1v4a1 1 0 102 0V6a1 1 0 00-1-1z"
    fillRule="evenodd"
  />
</svg>
`;
